### PR TITLE
Fixed delete repo button in add teams section

### LINKED ISSUE #462: Delete repo not working on create teams page
### Description:

Removal of a selected repo is not working on the create teams form in `/teams`

### Steps to reproduce:

1. Go to `/teams`
2. Click on `+ Add new team`
3. Type in and select some repos
4. Click on Delete button

### Expected behavior:

- a pointer cursor on hover to show click-ability
- removal of repo on clicking delete-button

### Actual behavior:

- normal cursor on hover
- removal doesn't work

### Server Setup Information:

- Operating System: 
- Deployment Method: 

### Additional context

https://github.com/middlewarehq/middleware/assets/76566992/4b25df5c-8ee3-40c6-b3d0-64a87bb507be

### Side Quest:

- on selecting a repo, there comes a table, with a selector showing: `PR Merge` and `Workflow`.
- on selecting `Workflow`, another dropdown pops up.
- this menu seems to have a very heavy padding, the wrapper containg the menu-items should have little to no padding.

image

### PR DESCRIPTION
**Pull request by ayush3160**
## Linked Issue(s) 
fixes #462 

## Proposed changes (including videos or screenshots)

https://github.com/middlewarehq/middleware/assets/89914602/0c4d33ec-c7de-427c-b481-3df218db1659

### CODE REVIEW

File: web-server/src/components/Teams/CreateTeams.tsx [hunk 1]
```diff
@@ -349,9 +350,12 @@ const DisplayRepos: FC = () => {
               <TableCell>
                 <FlexBox
                   title="Delete repo"
-                  sx={{ px: 1 }}
+                  sx={{ px: 1, cursor: 'pointer' }}
```
Comment (e-for-eshaan) [inline web-server/src/components/Teams/CreateTeams.tsx:353]: simply bass pointer as a boolean prop
```
<FlexBox
title="Delete repo"
pointer
sx..... blah blah
....
/>
```

File: web-server/src/components/WorkflowSelector.tsx [hunk 2]
```diff
@@ -126,14 +126,14 @@ export const DeploymentWorkflowSelector: FC<{ repo: BaseRepo }> = ({
         sx={{ textAlign: 'start' }}
       >
         {loading.value ? (
-          <FlexBox alignCenter gap2>
+          <FlexBox alignCenter gap2 sx={{ p: 2 }}>
```
Comment (e-for-eshaan) [inline web-server/src/components/WorkflowSelector.tsx:129]: can you show relevant snapshots/recordings for this change?

Comment (e-for-eshaan) [reply web-server/src/components/WorkflowSelector.tsx:129]: Refer this if it helps!
https://github.com/middlewarehq/middleware/blob/765c3d8654138b18e70b57c036f8da91bff8230a/web-server/src/components/TeamsList.tsx#L327-L333

**Review by e-for-eshaan — CHANGES_REQUESTED** [review]

File: web-server/src/components/Teams/CreateTeams.tsx (hunk 1 above)
Comment (jayantbh) [reply web-server/src/components/Teams/CreateTeams.tsx:353]: Like `<FlexBox ... pointer >...`

Comment (ayush3160) [reply web-server/src/components/Teams/CreateTeams.tsx]: Done

File: web-server/src/components/WorkflowSelector.tsx (hunk 2 above)
Comment (ayush3160) [reply web-server/src/components/WorkflowSelector.tsx]: Thanks for the reference , I have used the above method and this is how it looks now.

Screenshot (103)

**Review by e-for-eshaan — APPROVED** [review]
LGTM

**Pull request merged**